Library/Formula/boot2docker.rb

```diff
@@ -3,8 +3,8 @@ class Boot2docker < Formula
   homepage "https://github.com/boot2docker/boot2docker-cli"
   # Boot2docker and docker are generally updated at the same time.
   # Please update the version of docker too
-  url "https://github.com/boot2docker/boot2docker-cli.git", :tag => "v1.7.0",
-    :revision => "7d89508118ffbf5f3313cfa8a10d563942e3a643"
+  url "https://github.com/boot2docker/boot2docker-cli.git", :tag => "v1.7.1",
+    :revision => "c2027989c544a3320be7888cc197226a71dda1ba"
```

Comment:
The revision seems wrong:

```
$ git show -s v1.7.1
commit 8fdc6f573bf08149b6311681800d55fda6e19e71
Author: Tianon Gravi <admwiggin@gmail.com>
Date:   Tue Jul 14 16:50:53 2015 -0700

    Bump to 1.7.1
```

Also, why don’t we use the GitHub archive instead? https://github.com/boot2docker/boot2docker-cli/releases/tag/v1.7.1